app/demo/template/icons.htm

```diff
@@ -0,0 +1,33 @@
+<div id="icons">
+    <add-participants-icon></add-participants-icon>
+    <archive-icon></archive-icon>
+    <arrow-left-icon></arrow-left-icon>
+    <block-icon></block-icon>
+    <camera-icon></camera-icon>
+    <check-icon></check-icon>
+    <chevron-icon></chevron-icon>
+    <close-icon></close-icon>
+    <copy-icon></copy-icon>
+    <delete-icon></delete-icon>
+    <devices-icon></devices-icon>
+    <disclose-icon></disclose-icon>
+    <edit-icon></edit-icon>
+    <fullscreen-icon></fullscreen-icon>
+    <group-icon></group-icon>
+    <guest-icon></guest-icon>
+    <hangup-icon></hangup-icon>
+    <link-icon></link-icon>
+    <leave-icon></leave-icon>
+    <message-icon></message-icon>
+    <micoff-icon></micoff-icon>
+    <minus-icon></minus-icon>
+    <mute-icon></mute-icon>
+    <people-icon></people-icon>
+    <pickup-icon></pickup-icon>
+    <plus-icon></plus-icon>
+    <profile-icon></profile-icon>
+    <screenshare-icon></screenshare-icon>
+    <settings-icon></settings-icon>
+    <undo-icon></undo-icon>
+    <verified-icon class="override-color"></verified-icon>
+</div>
```

Comment:
Please add empty line at eof

Reply:
Do we have any header in the html files?

Reply:
These files don't have any headers as they are just fragments.